README.md

```diff
@@ -2,6 +2,10 @@
 
 # Kotlin/Native wrapper for the Godot Game Engine
 
+## Release README
```

Comment:
I would say it's more a `Release Documentations` or `Documentation for Releases`